pkgs/tools/misc/spigot/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchurl
+, gmp
+}:
+
+stdenv.mkDerivation rec {
+  name = "spigot-20200101";
+  src = fetchurl {
+    url = "https://www.chiark.greenend.org.uk/~sgtatham/spigot/${name}.b1b0b20.tar.gz";
+    sha256 = "0cf9h6x2n016ngy5qx53kkmgi4v8a3zinmh534j4gafmmzap061s";
+  };
+
+  configureFlags = [ "--with-gmp" ];
+
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
```

Comment:
Is this really necessary? It shouldn't be.

Reply:
I'm assuming it is the buildPackages.stdenv.cc that isn't necessary.  I wish I remember which nix package I played copy-paste monkey for this.